MarkdownToLatex/MarkdownToLatex/LatexRenderer.cs

```diff
@@ -12,35 +12,17 @@ public static class LatexRenderer
         /// <summary>a List containing the LaTeX document line by line.</summary>
         public static List<string> LatexLines;
 
-        /// <seealso cref="InList"/>
-        private static byte _inlist;
-
         /// <summary>Contains list level of last Markdown line ('0' if it was not in a list)</summary>
         private static byte oldInList;
 
         /// <summary>a byte indicating whether a list is currently rendered.</summary>
-        public static byte InList {get => _inlist; internal set {
-            if(value <= 3){
-                _inlist = value;
-            } else {
-                _inlist = 3; 
-            }
-        }}
-
-        /// <seealso cref="InQuote"/>
-        private static byte _inquote;
+        public static byte InList {get; internal set;}
```

Comment:
Jetzt kannst du die wieder auf `private` setzen :D

Reply:
Oh stimmt xD Wegen dem Problem oben (da kann ich irgendwie auf die Nachricht nicht antworten): Ich habe eigentlich in 6810871 dort ein `File.Create(path2).Close()` hinzugefügt, um die Datei zu erstellen und sofort wieder zu schließen (Damit der andere Test, der auf die Datei zugreift nicht rummeckert). Irgendwie hat GitHub das aber nicht richtig angezeigt. Ich hab jetzt mal in 00148f9 danach ne Leerzeile eingefügt, vielleicht zeigt er es jetzt an